src/spork.cpp

```diff
@@ -219,7 +218,7 @@ int64_t CSporkManager::GetSporkValue(SporkId nSporkID)
 {
     LOCK(cs);
 
-    int64_t nSporkValue = -1;
+    int64_t nSporkValue;
```

Comment:
No... this `nSporkValue` is only used in L222-224, and in SporkValueIsActive, nSporkValue is always set if SporkValueIsActive returns true. If SporkValueIsActive returns false, then this method returns a hard coded -1 on L232

Reply:
Ah, right. Well, this is confusing. Why don't we get rid of `SporkValueIsActive` and implement all its logic in `GetSporkValue` instead since it's not used anywhere else anyway? Thoughts?

Reply:
I don't have any problem with this, maybe in another pr or in another commit I'll implement later